services/checks/checks.go

```diff
@@ -416,6 +417,111 @@ func (s *Service) EnableChecks(checkNames []string) error {
 	return nil
 }
 
+// ChangeInterval changes a check's interval to the value received from the UI.
+func (s *Service) ChangeInterval(checkName string, interval managementpb.SecurityCheckInterval) error {
+	checks := s.GetAllChecks()
+	if len(checks) == 0 {
+		return errors.New("no checks loaded")
+	}
+
+	for _, c := range checks {
+		if c.Name == checkName {
```

Comment:
I prefer negate condition and continue. So code below is not nested.

Reply:
done thanks